addons/stock/views/stock_quant_views.xml

```diff
@@ -67,6 +67,11 @@
                                 <field name="inventory_quantity"/>
```

Comment:
In addition of your diff, this shouldn't be replaced by the field `quantity` instead ?
We display the label "*Quantity On Hand*" which is the label of the field `quantity`, but we use the field `inventory_quantity`, which seems wrong (cc. @yhu-odoo )

Reply:
yes I agree, but maybe we should also add the "counted quantity" information which corresponds to the `inventory_quantity`, and the on hand quantity will be used for the `quantity` field?